tests/e2e-pw/bin/test-env-setup.sh

```diff
@@ -11,6 +11,7 @@ script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 ###################################################################################################
 
 wp-env run tests-cli "wp site empty --yes"
+wp-env run tests-cli "wp plugin activate woocommerce-blocks"
```

Comment:
Perhaps instead of this, we could just activate it [like this](https://github.com/woocommerce/woocommerce-blocks/pull/9500/files#diff-9fd21b09a0144355cfd001e3952767df08b2e04241994470aa1d627f797b171eR8)

Reply:
Good point! Fixed with 011c02b9781104c5bcab962708e0bccf00fa18c7